packages/itwinui-react/src/core/Table/filters/DateRangeFilter/DatePickerInput.tsx

```diff
@@ -24,6 +25,7 @@ export type DatePickerInputProps = {
    * The 'to' date for the 'from' DatePickerInput or the 'from' date for the 'to' DatePickerInput
    */
   selectedDate?: Date;
+  localizedNames?: DatePickerLocalizedNames;
```

Comment:
It would be nice to add jsdocs to props. localizedNames is rather generic and might need a comment to understand where it is used.

Reply:
Added